aries-site/src/data/structures/templates.js

```diff
@@ -14,9 +19,14 @@ export const templates = [
         fit: 'contain',
       },
     },
+    relatedContent: ['Dashboards', 'Page Layouts'],
```

Comment:
Do we want to add in 'Cards' as well? Will be 'Coming Soon' content immediately, but should be filled in in the next week or so.

Reply:
Yes, let's add Cards.